.github/workflows/nightly-build.yml

```diff
@@ -0,0 +1,43 @@
+name: Upload Nightly Binaries
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # run at mid night
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    name: Upload Nightly Binaries
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
+
+      - name: Install rpm
+        run: sudo apt install rpm
```

Comment:
I suggest adding a comment that `sudo apt install rpm` is done so the Ubuntu host can create both the DEB and RPM install packages.

Also, can you confirm that the RPM that gets built by this Ubuntu machine can be installed on a AL2 (or RedHat or CentOS) host?